src/jit/compphases.h

```diff
@@ -100,6 +100,7 @@ CompPhaseNameMacro(PHASE_CLR_API,                "CLR API calls",
 #endif
 
 CompPhaseNameMacro(PHASE_EMPTY_FINALLY,          "Remove empty finally",           "EMPTYFIN",  false, -1)
+CompPhaseNameMacro(PHASE_EMPTY_TRY,              "Remove empty try",               "EMPTYTRY",  false, -1)
 CompPhaseNameMacro(PHASE_CLONE_FINALLY,          "Clone finally",                  "CLONEFIN",  false, -1)
```

Comment:
nit: since you made empty try removal execute first, maybe you can list it first here, too. (and in the function declarations)